tests/performance_tests/CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ configure_file(genesis.json genesis.json COPYONLY)
 configure_file(validate_nodeos_plugin_args.py validate_nodeos_plugin_args.py COPYONLY)
 
 add_test(NAME performance_test_basic COMMAND tests/performance_tests/performance_test_basic.py -v -p 1 -n 1 --target-tps 20 --tps-limit-per-generator 10 --clean-run WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
+add_test(NAME performance_test_basic_ex_trx_spec COMMAND tests/performance_tests/performance_test_basic.py -v -p 1 -n 1 --target-tps 20 --tps-limit-per-generator 10 --clean-run --exercise-trx-specification WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
```

Comment:
I was expecting to see data passed in instead of `--exercise-trx-specification`

Reply:
Addressed in updates, most recently: [777d9d3](https://github.com/AntelopeIO/leap/pull/626/commits/777d9d30bf5918a4fb6f106d45eef3f69d3b59f4)